orbit/radius.py

```diff
@@ -462,14 +462,17 @@ def form_respond():
     if not (student_id := rocket.body_args_query('student_id')):
         rocket.msg('you must provide a student id')
         return form_respond()
-    if not (registration_data := db.reg_get_and_del_by_stuid(student_id)[0]):
+    query = (db.Registration
+             .delete()
+             .where(db.Registration.student_id == student_id)
+             .returning(db.Registration))
+    if not (registration := next(iter(query.execute()), None)):
```

Comment:
same comment as session drop before, why not use a short helper function to clean this up?

Reply:
I think it actually is pretty clean and idiomatic python. I don't want to make a helper because I think that it doesn't cross the threshold to where the inconvenience of have having to go and look up the helper is worth it for making the code shorter especially given that we only do it twice.